ar/publish.css

```diff
@@ -10,15 +10,16 @@ pre {
 	padding: 32px 18px 0 0;
 }
 .site-body-left-column-site-name {
-	padding: 4px 0 32px 32px;
+	/* padding: 4px 0 32px 32px; */
```

Comment:
Do you still need this? Would prefer to delete commented out code if no longer needed

Reply:
I don't need it if we replace the text with this svg, just like the English doc; if not, then I need it.

![image](https://github.com/obsidianmd/obsidian-help/assets/128622475/1c782d11-9552-4758-be6f-91a673ecd1a9)
I will add the svg to the attachments folder.